pkg/report/writer.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"golang.org/x/xerrors"
 
+	fanalTypes "github.com/aquasecurity/fanal/types"
```

Comment:
nit: in other packages, we use `ftypes`.

Reply:
@knqyf263 Done